src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,40 @@
+package duke;
+
+public class Deadline extends Task {
+
+  private String date;
+
+  public Deadline(String description, String date) {
+    this.description = description;
+    this.date = date;
+    this.done = false;
+  }
```

Comment:
There is some redundancy here, try this instead:
```suggestion
  public Deadline(String description, String date) {
    this(description, date, false)
  }
```